pkg/arm/data/etc/etcd/ca.crt

```diff
@@ -0,0 +1 @@
+{{ CertAsBytes .Config.Certificates.EtcdCa.Cert | String }}
```

Comment:
follow-up: path pkg/arm/data/etc will need to be renamed pkg/arm/data/master/etc so that we can add pkg/arm/data/node for node startup.